include/vcpkg/base/parallel-algorithms.h

```diff
@@ -29,55 +30,55 @@ namespace vcpkg
         }
     }
 
-    template<class RanIt, class F>
-    void parallel_for_each_n(RanIt begin, size_t work_count, F cb) noexcept
+    template<class T, class F>
+    void parallel_for_each(View<T> view, F cb) noexcept
```

Comment:
Billy wanted this to be called `parallel_for_each_n`

Reply:
The `_n` part is due to the interface matching `std::for_each_n`. If we are OK with this change overall I think the name change is OK.